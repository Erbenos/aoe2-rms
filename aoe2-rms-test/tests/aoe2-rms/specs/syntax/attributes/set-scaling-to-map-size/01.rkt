#lang aoe2-rms
; Create clumps of 10 gold and scale the number of groups to map size.

<OBJECTS-GENERATION>
(create-object 'GOLD
               (number-of-objects 10)
               (number-of-groups 3)
               (set-scaling-to-map-size)
               (set-tight-grouping))
