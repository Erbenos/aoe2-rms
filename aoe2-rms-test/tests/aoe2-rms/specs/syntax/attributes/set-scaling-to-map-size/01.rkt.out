<OBJECTS_GENERATION>
create_object GOLD {
  number_of_objects 10
  number_of_groups 3
  set_scaling_to_map_size
  set_tight_grouping
}
